lib/python/qmk/cli/kle2json.py

```diff
@@ -69,7 +69,7 @@ def kle2json(cli):
     # Replace layout in keyboard json
     keyboard = keyboard.replace('"LAYOUT_JSON_HERE"', layout)
     # Write our info.json
-    file = open(out_path + "/info.json", "w")
+    file = open(str(out_path) + "/info.json", "w")
```

Comment:
```suggestion
    file = open(out_path / "info.json", "w")
```

Looks like this got missed in the 3.5 -> 3.6 transition. :)